### PR TITLE
Fix/improvelinecoverage

### PR DESCRIPTION
**Pull request by wroel**
Made tests for the ViewData class and its subclasses.

### CODE REVIEW

File: src/test/java/nl/tudelft/ti2206/semgr8/model/GameTest.java [hunk 1]
```diff
@@ -109,4 +109,38 @@ public void testIncreaseLevel() {
...
+    }
+
+    /**
+     * Test if getScore returns the right score.
+     */
+    @Test
+    public void testGetScore() {
+        assertEquals(score, game.getScore());
+    }
 }
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/GameTest.java]: missing newline

**Comment by albert-smit** [comment]
rest looks good

File: src/test/java/nl/tudelft/ti2206/semgr8/model/data/SelectDataTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+
+import nl.tudelft.ti2206.semgr8.model.board.Position;
+import org.junit.Before;
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Test class for the SelectData class
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/data/SelectDataTest.java]: checkstyle: missing period

File: src/test/java/nl/tudelft/ti2206/semgr8/model/data/SwapDataTest.java [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Test class for the SwapData class.
+ */
+@SuppressWarnings("checkstyle:magicnumber")
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/data/SwapDataTest.java]: PMD: too many methods

File: src/test/java/nl/tudelft/ti2206/semgr8/model/data/SwapDataTest.java [hunk 4]
```diff
@@ -0,0 +1,135 @@
...
+        assertEquals("SwapData{gem1=Gem{color=0, pos=Position{x=0, y=0}}, "
+                + "gem2=Gem{color=1, pos=Position{x=1, y=0}}}", swapData.toString());
+    }
+
+    /**
+     * Test if equals method finds two the same SwapData to be equal.
+     */
+    @Test
+    public void testEqualsWhenTheSame() {
+        assertTrue(swapData.equals(swapData));
```
Comment (albert-smit) [inline src/test/java/nl/tudelft/ti2206/semgr8/model/data/SwapDataTest.java]: assertEquals

**Comment by albert-smit** [comment]
rest looks good

**Comment by kiliangrashoff** [comment]
Processed PR comments, no further code changes, only documentation.

**Pull request merged**